plugin/src/main/java/net/dzikoysk/funnyguilds/shared/TimeDivision.java

```diff
@@ -0,0 +1,96 @@
+package net.dzikoysk.funnyguilds.shared;
+
+import java.util.Arrays;
+
+public enum TimeDivision {
+
+    SECOND(1000L, "sekunda", "sekundę", "sekundy", "sekund", "s", "s", "sek"),
+    MINUTE(60000L, "minuta", "minutę", "minuty", "minut", "m", "m", "min"),
+    HOUR(3600000L, "godzina", "godzinę", "godziny", "godzin", "h", "h", "godz"),
+    DAY(86400000L, "dzień", "dzień", "dni", "dni", "d", "d", "dni", "day"),
+    WEEK(604800000L, "tydzień, ", "tydzień", "tygodnie", "tygodni", "w", "w", "t", "tyg"),
+    MONTH(2592000000L, "miesiąc", "miesiąc", "miesiące", "miesięcy", "ms", "mo", "ms", "mies"),
+    YEAR(31536000000L, "rok", "rok", "lata", "lata", "r", "y", "r", "l", "lat", "rok");
```

Comment:
Nie no tak to się nie bawimy, że po polsku hardcoded w enumie. Cała ta klasa to jakiś potworek ogółem.😬

Reply:
UWAGA, PODAJĘ ROZWIĄZANIE: https://github.com/OkaeriPoland/okaeri-pluralize

Reply:
UWAGA, PODAJĘ JESZCZE LEPSZE ROZWIĄZANIE: https://github.com/OkaeriPoland/okaeri-placeholders/issues?q=is%3Aissue+is%3Aclosed

Reply:
Maybe in #2103

Reply:
No to chociaż okaeri-pluralize (to jedna klasa jest przecież) i do tego config tych śmiesznych stringów zamiast enuma.

Reply:
No dobra, tylko teraz trzeba wymyślić jakiś sensowny format konfiguracji co przy okazji różne Locale wspiera

Reply:
Dobra nie wiem jak ogarnąć przypadki w sensowny sposób ale jest przynajmniej konfiguracja